docker-compose.yml

```diff
@@ -26,18 +26,18 @@ services:
     volumes:
       - './django_celery_beat/:/app/django_celery_beat/'
 
-  celery-beat:
+  celery:
     depends_on:
-      - base
+      - django
       - postgres
       - rabbit
     build:
       context: .
-      dockerfile: docker/celery-beat/Dockerfile
-    entrypoint: ["/app/docker/celery-beat/entrypoint.sh"]
+      dockerfile: docker/celery/Dockerfile
+    entrypoint: ["/app/docker/celery/entrypoint.sh"]
     environment:
         CELERY_BROKER_URL: 'amqp://guest:guest@rabbit:5672'
-    command: ["python3", '-m', "celery", "-A", "mysite", "beat", "-l", "info", "--scheduler", "django_celery_beat.schedulers:DatabaseScheduler"]
+    command: ["python3", '-m', "celery", "-A", "mysite", "worker", "--beat", "-l", "info", "--scheduler", "django_celery_beat.schedulers:DatabaseScheduler"]
```

Comment:
Should `beat` be changed to `worker` for the whole project? Does this need a re-merge of master?

Reply:
The docker setup is to have a quick way of testing the project, mostly as a means of showcasing the admin panel. However if you try doing any testing in the docker container it won't work without a celery worker. It doesn't have any impact on the library itself. I just kept the change around because it was quite confusing trying to figure out why none of the manual tests were working.